Gems/ScriptCanvasDeveloper/Code/Editor/Source/EditorAutomation/EditorAutomationActions/ScriptCanvasActions/GraphActions.cpp

```diff
@@ -134,7 +134,8 @@ namespace ScriptCanvas::Developer
 
     void CreateNewFunctionAction::OnActionsComplete()
     {
-        m_graphId = m_newGraphAction->GetGraphId();
+        if (m_newGraphAction)
+            m_graphId = m_newGraphAction->GetGraphId();
```

Comment:
By convention in O3DE the body of an if is always in brackets:
```suggestion
        if (m_newGraphAction)
        {
            m_graphId = m_newGraphAction->GetGraphId();
        }
```

Reply:
oh yes sry! thanks :)